src/Table/Table.stories.tsx

```diff
@@ -33,12 +34,14 @@ Default.args = {
     ['2011-01-01T00:00:00.000Z', 'example-2', <button>button</button>],
     ['2012-01-01T00:00:00.000Z', 'example-3', <h3>heading #3</h3>],
   ],
+  action: (item) => console.log(item),
```

Comment:
Do we want console logs in the final thing or are you adding this to prevent the linter from showing red lines ?

Reply:
it's in the story so it's not like it's exported. My preference would be to use `action` like we do elsewhere unless there's a good reason not to. @n3op2 ?

Reply:
oops, it was me testing:) yes, should have been an action, did not update the default variant